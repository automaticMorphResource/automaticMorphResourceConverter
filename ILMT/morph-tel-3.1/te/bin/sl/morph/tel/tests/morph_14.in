1	aMxukane	unk	
1	AstreliyA	unk	
1	praBuwvaM	unk	
1	kvIns	unk	
1	lAMd	unk	
1	praBuwva	unk	
1	sahakAraMwo	unk	
1	I	unk	
1	prAMwAnni	unk
1	aMwatinI	unk	
1	parirakRiwa	unk	
1	prAMwaMgA	unk	
1	prakatiMci	unk	
1	aneka	unk	
1	rakRaNa	unk	
1	caryalanu	unk	
1	cepaduwoMxi	unk	
1	.	unk	
