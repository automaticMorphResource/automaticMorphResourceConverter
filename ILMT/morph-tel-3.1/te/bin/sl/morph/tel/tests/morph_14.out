1	aMxukane	unk	<fs af='aMxukane,avy,,,,,,'>
2	AstreliyA	unk	<fs af='AstreliyA,n,,sg,,d,0,0'>|<fs af='AstreliyA,n,,sg,,o,ti,ti'>
3	praBuwvaM	unk	<fs af='praBuwvaM,n,,sg,,d,0,0'>
4	kvIns	unk	<fs af='kvIns,n,,sg,,d,0,0'>|<fs af='kvIns,n,,sg,,o,ti,ti'>
5	lAMd	unk	<fs af='lAMd,unk,,,,,,'>
6	praBuwva	unk	<fs af='praBuwvaM,n,,sg,,o,ti,ti'>
7	sahakAraMwo	unk	<fs af='sahakAraM,n,,sg,,,wo,wo'>
8	I	unk	<fs af='I,avy,,,,,,'>|<fs af='I,avy,,,,,,'>
9	prAMwAnni	unk	<fs af='prAMwaM,n,,sg,,,nu,nu'>|<fs af='prAMwaM,n,,sg,,,ni,ni'>
10	aMwatinI	unk	<fs af='aMwatinI,unk,,,,,,'>
11	parirakRiwa	unk	<fs af='parirakRiwaM,n,,sg,,o,ti,ti'>
12	prAMwaMgA	unk	<fs af='prAMwaM,n,,sg,,,gA,gA'>
13	prakatiMci	unk	<fs af='prakatiMcu,v,any,any,any,,i,i'>|<fs af='prakatiMcu,v,any,any,any,,i,i'>
14	aneka	unk	<fs af='aneka,adj,,,,,0,0_adj'>|<fs af='anekaM,n,,sg,,o,ti,ti'>|<fs af='anekudu,n,,sg,,o,ti,ti'>
15	rakRaNa	unk	<fs af='rakRaNa,n,,sg,,d,0,0'>|<fs af='rakRaNa,n,,sg,,o,ti,ti'>
16	caryalanu	unk	<fs af='carya,n,,pl,,,ni,ni'>
17	cepaduwoMxi	unk	<fs af='cepattu,v,fn,sg,3,,wunn,wunn'>
18	.	unk	<fs af='&dot;,punc,,,,,,'>
